invenio_rdm_records/fixtures/vocabularies.py

```diff
@@ -84,29 +84,41 @@ def __iter__(self):
 class VocabulariesFixture:
     """Vocabularies fixture."""
 
-    def __init__(self, identity, search_path, filename):
+    def __init__(self, identity, search_paths, filename):
         """Initialize the fixture."""
-        self._search_path = search_path
+        self._search_paths = search_paths
         self._filename = filename
         self._identity = identity
 
     def load(self):
-        """Load the fixture."""
-        with open(self._search_path.path(self._filename)) as fp:
-            data = yaml.load(fp)
-            for id_, entry in data.items():
-                self.load_vocabulary(id_, entry)
+        """Load the fixture.
+
+        Fixtures found sooner in self._search_paths have priority.
+        """
+        ids = set()
+
+        for path in self._search_paths:
+            filepath = path / self._filename
+
+            # Providing a vocabularies.yaml file is optional
+            if not filepath.exists():
+                continue
+
+            with open(filepath) as fp:
+                data = yaml.safe_load(fp)
+                for id_, entry in data.items():
+                    if id_ not in ids:
+                        entry["data-file"] = path / entry["data-file"]
+                        self.load_vocabulary(id_, entry)
+                        ids.add(id_)
```

Comment:
minor: should we instead fail if there are duplicated? or at least issue a warning so the user might check it?

Reply:
The `ids` here refer to the ids found in the vocabularies.yaml file -not the id of each individual vocabulary record. There are 2 layers of files for vocabularies. 

We allow overridding vocabularies on a per vocabulary basis. That is, an instance can provide a `vocabularies.yaml` that lists only the `languages` id (that entry points to a separate file with all the actual languages entries). The instance will load those languages, but continue looking for any other vocabularies in the next location where it could find a `vocabularies.yaml` file (the one provided with this package). There, it will load the `vocabularies.yaml` but it won't load the `languages` vocabulary this time around, because it already did. This is the intended way to allow load priority.